hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/endpointauthentication/WebAuthenticationUtils.java

```diff
@@ -0,0 +1,39 @@
+package ca.bc.gov.hlth.hnsecure.endpointauthentication;
```

Comment:
Package name is maybe a little verbose? Could just put until a generic util package? Not too concerned either way.